jacodb-ets/src/main/kotlin/org/jacodb/ets/model/EtsSignature.kt

```diff
@@ -31,20 +31,37 @@ data class EtsFileSignature(
     override fun toString(): String {
         // Remove ".d.ts" and ".ts" file ext:
         val tmp = fileName.replace(REGEX_TS_SUFFIX, "")
-        return "@$projectName/$tmp"
+        // return if (projectName.isNotBlank()) {
+        //     "@$projectName/$tmp"
+        // } else {
+        //     tmp
+        // }
+        return tmp
+    }
+
+    override fun equals(other: Any?): Boolean {
+        if (this === other) return true
+        if (javaClass != other?.javaClass) return false
+
+        other as EtsFileSignature
+        if (fileName != other.fileName) return false
```

Comment:
Checking `projectName` was omitted deliberately, since I do not trust such loose meta-information to be used for equality check. Probably a clarifying comment is necessary in this case.